packages/manager/modules/advices/src/advices/index.less

```diff
@@ -16,11 +16,11 @@ ovh-advices {
   }
 
   oui-tile-definition.oui-tile__item {
-    border-top: 1px solid #ffffff;
+    border-top: 1px solid #aee5f3;
```

Comment:
Don't know if it has been already discuss, but might be interesting to move this into `ovh/ovh-ui-kit`.
What do you think about it?

Reply:
Good suggestion. ovh-ui-kit has no utility for border. It has utility classes for background and text colour. Adding utility just for this does not justify the efforts. Shall we consider this once we have multiple use cases?

Reply:
Bootstrap4 on the other hand provides border utilities.

Reply:
Yes it has border-primary, border-secondary etc. No class with color "aee5f3". Did you know any solution/workaround to change the color?

Reply:
Maybe guidelines around both tile/box components can/should evolve to handle such use case.